src/strategy.ts

```diff
@@ -188,34 +186,27 @@ export class Strategy extends OAuth2Strategy {
     });
   }
 
-  /**
-   * Authenticate request by delegating to a service provider using OAuth 2.0.
-   */
-  authenticate(req: Request, options?: AuthenticateOptions) {
-    options = options || {};
+  addDefaultScopes(scopes: string[] | Array<string>, options: StrategyOptions) {
```

Comment:
question: Maybe my TypeScript-fu is weak, but isn't `string[]` and `Array<string>` equivalent?

Reply:
🤦 yes it is